src/components/CommentForm.vue

```diff
@@ -0,0 +1,89 @@
+<script setup>
+import { ref, onMounted, watch } from 'vue'
+import { addComment } from '@/api/comments'
+import InputField from './InputField.vue'
+import TextAreaField from './TextAreaField.vue'
+
+const { postId, formName, formEmail } = defineProps({
+  postId: Number,
+  formName: String,
+  formEmail: String
+})
+
+const emits = defineEmits(['closeForm', 'addComment'])
+
+const name = ref('')
+const email = ref('')
+const body = ref('')
+
+const errors = ref({
+  name: '',
+  email: '',
+  body: ''
+})
+
+const isLoading = ref(false)
+
+onMounted(() => {
+  name.value = formName
+  email.value = formEmail
+})
+
+watch(name, () => {
+  errors.value.name = ''
+})
+
+watch(email, () => {
+  errors.value.email = ''
+})
+
+watch(body, () => {
+  errors.value.body = ''
+})
```

Comment:
You can consolidate it into one watcher 

watch([name, email, body], ([newName, newEmail, newBody]) => {
  if (newName) errors.value.name = ''
  if (newEmail) errors.value.email = ''
  if (newBody) errors.value.body = ''
})